.docker/test.ini

```diff
@@ -40,7 +40,7 @@ beaker.session.secret = bSmgPpaxg2M+ZRes3u1TXwIcE
 app_instance_uuid = 6e3daf8e-1c6b-443b-911f-c7ab4c5f9605
 
 # repoze.who config
-who.config_file = %(here)s/who.ini
+who.config_file = /srv/app/who.ini
```

Comment:
I would prefer to leave this as a variable, rather than relying on the Docker container path.